hordak/models/core.py

```diff
@@ -520,29 +542,80 @@ class Leg(models.Model):
     objects = CustomLegManager()
 
     def __str__(self):
-        return f"{self.type.title()} {self.account.name} ({self.account.full_code}) {self.amount}"
+        return (
+            f"{self.type.title()} {self.account.name} "
+            f"({self.account.full_code}) {self.amount} {self.type_short}"
+        )
+
+    def __init__(self, *args, amount: Money = None, **kwargs):
+        if amount is not None:
+            warnings.warn(
+                "Specifying `amount` when creating an Account is deprecated. "
```

Comment:
I think this is supposed to say `... when creating a Leg...` not Account?

Reply:
Ooh yes, great catch! Fixed.